index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title>JS Quiz</title>
+    </head>
+    <body style="text-align: center; vertical-align: middle; margin-top: 15%;">
+        <h1>Input size of tree to create:</h1>
+        <form id="form" action="index.html" method="post">
+            <input type="text" name="height" id="height" value="" placeholder="amout of rows" onkeydown="checkKey(event)">
```

Comment:
No running javascript from your html. You can achieve the same result by coding an event listener in your js file.